src/app/services/movie.ts

```diff
@@ -0,0 +1,43 @@
+export class Movie {
```

Comment:
You use this class as interface (you don't create Movies instances `new Movie(...)`), it is needed just for typings, so you should replace this definition to interface definition